Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetConnection.tsx

```diff
@@ -181,9 +197,8 @@ function AssetConnectionWindow(props: { Name: string, ID: number, TypeID: number
         });
     }
 
-
     function handleSelect(item) {
-        history.push({ pathname: homePath + 'index.cshtml', search: '?name=Asset&AssetID=' + item.row.AssetID})
+        history.push({ pathname: homePath + 'index.cshtml', search: '?name=Asset&AssetID=' + item.row.AssetID })
```

Comment:
@gcsantos-gpa I bet this conflicts with #606 ?

Reply:
It would yes